src/_locales/ru/messages.json

```diff
@@ -91,45 +91,45 @@
     "html_options_suspend_screen_capturing_screen_only": { "message": "Захватывать только верх экрана" },
     "html_options_suspend_screen_capturing_entire_page": { "message": "Захватывать всю страницу" },
     "html_options_suspend_screen_capturing_tooltip_line1a": {"message": "На заблокированной вкладке расширение может отображать предварительный просмотр изображения вкладки"},
```

Comment:
better would be if we simplify here a bit: "На приостановленной вкладке расширение может отображать скриншот вкладки"

Reply:
I changed this translation, why do not you see the final version?

Reply:
my version is
На приостановленной вкладке расширение будет отображать скриншот **страницы**
because
![image](https://user-images.githubusercontent.com/23438485/52139224-96c94700-2660-11e9-916e-878e3b886f86.png)

Reply:
Can see now, thanks, urs current variant better